templates/status.html

```diff
@@ -86,6 +89,49 @@
                 request_api(website);
             });
         }
+
+        function constructLogoElement (website, logos) {
+            // defaults to font awesome icons
+            var logoElement = {
+                attributeValue: `fa fa-${website}`,
+                htmlAttribute: 'class',
+                htmlElement: 'i',
+            };
+
+            if (logos[website]) {
+                // the key defined in the yml. e.g., url, fontawesome
+                var ymlKey = Object.keys(logos[website])[0];
+                // determines html attribute to be used to display icon. e.g., src, class
+                logoElement.htmlAttribute = determineHtmlAttribute(ymlKey);
+                // determines html element to be used based off of the attribute. e.g., i, img
+                logoElement.htmlElement = determineHtmlElement(logoElement.htmlAttribute);
+                // gets the attribute value from the yml file.
+                logoElement.attributeValue = logos[website][ymlKey];
+            }
+            return logoElement;
+        }
+
+        function determineHtmlElement(htmlAttribute) {
+            var element;
+            if (htmlAttribute === 'src') {
+                element = 'img';
+            } else if (htmlAttribute === 'class') {
+                element = 'i';
+            }
+            return element;
+        }
+
+        function determineHtmlAttribute(key) {
```

Comment:
Should have more descriptive name (include logo in the name of the function), or add comment to explain what is the function for.